packages/compartment-mapper/src/policy.js

```diff
@@ -113,29 +113,6 @@ export const dependencyAllowedByPolicy = (namingKit, packagePolicy) => {
   return !!policyLookupHelper(packagePolicy, 'packages', canonicalName);
 };
 
-const validateDependencies = (policy, canonicalName) => {
```

Comment:
all this is no longer necessary as this was alerting early about missing policy entries for known dependencies.